dev/com.ibm.ws.messaging.comms.client/src/com/ibm/ws/sib/jfapchannel/impl/CommsOutboundChain.java

```diff
@@ -127,89 +159,130 @@ void unbindSecureFacet(OutboundSecureFacet oldFacet) {
 
-            ChannelData tcpChannel = cfw.getChannel(tcpChannelName);
+                ChannelData tcpChannel = cfw.getChannel(tcpChannelName);
 
-            if (tcpChannel == null) {
-                String typeName = (String) tcpOptions.get("type");
-                tcpChannel = cfw.addChannel(tcpChannelName, cfw.lookupFactory(typeName), new HashMap<Object, Object>(tcpOptions));
-            }
-
-            ChannelData jfapChannel = cfw.getChannel(jfapChannelName);
-            if (jfapChannel == null)
-                jfapChannel = cfw.addChannel(jfapChannelName, cfw.lookupFactory("JFapChannelOutbound"), null);
+                if (tcpChannel == null) {
+                    String typeName = (String) tcpOptions.get("type");
+                    tcpChannel = cfw.addChannel(tcpChannelName, cfw.lookupFactory(typeName), new HashMap<Object, Object>(tcpOptions));
+                }
 
-            final String[] chanList = { jfapChannelName, tcpChannelName };
+                ChannelData jfapChannel = cfw.getChannel(jfapChannelName);
+                if (jfapChannel == null)
+                    jfapChannel = cfw.addChannel(jfapChannelName, cfw.lookupFactory("JFapChannelOutbound"), null);
 
-            ChainData cd = cfw.addChain(chainName, FlowType.OUTBOUND, chanList);
-            cd.setEnabled(true);
+                final String[] chanList = { jfapChannelName, tcpChannelName };
 
-            // The Fat test:
-            // /com.ibm.ws.messaging_fat/fat/src/com/ibm/ws/messaging/fat/CommsWithSSL/CommsWithSSLTest.java
-            // Checks for the presence of this string in the trace log, in order to ascertain that the chain has been enabled.
-            if (isAnyTracingEnabled() && tc.isDebugEnabled()) debug(this, tc, "JFAP Outbound chain"+ chainName + " successfully started ");
+                ChainData cd = cfw.addChain(chainName, FlowType.OUTBOUND, chanList);
+                cd.setEnabled(true);
 
-        } catch (ChannelException | ChainException exception) {
-            if (isAnyTracingEnabled() && tc.isDebugEnabled()) debug(this, tc, "JFAP Outbound chain " + chainName + " failed to start, exception ="+exception);
+                // The Fat test:
+                // /com.ibm.ws.messaging_fat/fat/src/com/ibm/ws/messaging/fat/CommsWithSSL/CommsWithSSLTest.java
+                // Checks for the presence of this string in the trace log, in order to ascertain that the chain has been enabled.
+                if (isAnyTracingEnabled() && tc.isDebugEnabled()) debug(this, tc, "JFAP Outbound chain"+ chainName + " successfully started ");
+                chainList.put(chainName, this);
+            } catch (ChannelException | ChainException exception) {
+                if (isAnyTracingEnabled() && tc.isDebugEnabled()) debug(this, tc, "JFAP Outbound chain " + chainName + " failed to start, exception ="+exception);
+            }
+        }else {
+        	// Use Netty Framework for transport
+            chainList.put(chainName, this);
```

Comment:
In this case they are meant to be the same but can't move the code out of the if/else block to be common because of the try catch block while setting up channelfw which would not add to the chainlist if `ChannelException|ChainException` occurs